core/src/main/java/hudson/tasks/UserNameResolver.java

```diff
@@ -58,7 +58,7 @@
      *
      * <p>
      * When multiple resolvers are installed, they are consulted in order and
-     * the search will be over when a name is found by someoene.
+     * the search will be over when a name is found by someene.
```

Comment:
Sorry, this is wrong (the fixes are all manual, so clearly my cursor was off by one when I suggested the correction).